salt/cli/caller.py

```diff
@@ -208,8 +211,29 @@ def call(self):
                     'Do you have permissions to '
                     'write to {0} ?\n'.format(proc_fn))
             func = self.minion.functions[fun]
+            data = {
+              'arg': args,
+              'fun': fun
+            }
+            data.update(kwargs)
+            executors = getattr(self.minion, 'module_executors', []) or \
+                        self.opts.get('module_executors', ['direct_call'])
```

Comment:
It can be cleaner:
```python
executors = getattr(self.minion, 'module_executors',
                    self.opts.get('module_executors', ['direct_call']))
```

Reply:
Same as https://github.com/saltstack/salt/pull/48344#discussion_r198911853